.vscode/tasks.recommended.json

```diff
@@ -0,0 +1,46 @@
+// This file contains recommended task for this project.
+// You can copy this file as "tasks.json" to make it work.
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
```

Comment:
plz update `caveats.md`

Reply:
what?

Reply:
explain why need to run `tsc --watch` and vscode shorthand task.

Reply:
The developer no need to worry about this. Most of the tasks are doing it behind the scene.

Reply:
there are no workflow changes in this PR. When starting webpack, a `tsc -w -b` will be auto started if there is no running instance.